sklearn/linear_model/logistic.py

```diff
@@ -1568,6 +1577,16 @@ class LogisticRegressionCV(LogisticRegression, BaseEstimator,
         Actual number of iterations for all classes, folds and Cs.
         In the binary or multinomial cases, the first dimension is equal to 1.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import load_iris
+    >>> from sklearn.linear_model import LogisticRegressionCV
+    >>> X, y = load_iris(return_X_y=True)
+    >>> clf = LogisticRegressionCV(cv=5,
+    ...         random_state=0).fit(X, y)
+    >>> clf.score(X, y) # doctest: +ELLIPSIS
```

Comment:
It seems strange that LogisticRegressionCV get worse result than LogisticRegression.
Maybe we should use cv=3 here since we don't have enough data. Not sure whether others will agree with me.